packages/docs/pages/users/fees.mdx

```diff
@@ -115,22 +115,17 @@ namadac transfer \
   --token NAM \
   --amount 10 \
   --signing-keys key1 \
-  --gas-limit 150000 \
+  --gas-limit 15000 \
   --gas-price 0.01
 ```
 
 This **might** incentivise validators to prioritise this transaction above those with a lower gas price.
 
 ## Paying fees with tokens in the MASP
 
-It is also possible to pay for fees using the MASP when dealing with a transfer transaction involving shielded inputs (shielding, shielded, and unshielding transfers both natively and with IBC).
-The purpose of this is to ensure that a user can make transactions on-chain even if they may not have NAM in their transparent balance.
-This is yet another incentive for users to keep the maximum amount of assets in the MASP.
-On top of this, paying fees via the MASP is required to prevent information leakage that could defy the purpose of the shielded pool.
+It is also possible to pay for fees using the MASP when dealing with a transfer transaction involving shielded inputs (shielded, and unshielding transfers both natively and with IBC): this is required to prevent information leakage that could defy the purpose of the shielded pool.
```

Comment:
```suggestion
It is also possible to pay for fees using the MASP when dealing with a transfer transaction involving shielded inputs (shielded, and unshielding transfers both natively and with IBC): this is required to prevent information leakage which defeats the purpose of the shielded pool.
```